src/main/java/it/gov/pagopa/payhub/activities/mapper/ingestionflow/debtposition/InstallmentSynchronizeMapper.java

```diff
@@ -0,0 +1,107 @@
+                .entityType(InstallmentSynchronizeDTO.EntityTypeEnum.valueOf(installmentIngestionFlowFileDTO.getEntityType().name()))
+                .fiscalCode(installmentIngestionFlowFileDTO.getFiscalCode())
+                .fullName(installmentIngestionFlowFileDTO.getFullName())
+                .address(installmentIngestionFlowFileDTO.getAddress())
+                .civic(installmentIngestionFlowFileDTO.getCivic())
+                .postalCode(installmentIngestionFlowFileDTO.getPostalCode())
+                .location(installmentIngestionFlowFileDTO.getLocation())
+                .province(installmentIngestionFlowFileDTO.getProvince())
+                .nation(installmentIngestionFlowFileDTO.getNation())
+                .email(installmentIngestionFlowFileDTO.getEmail())
+                .dueDate(installmentIngestionFlowFileDTO.getDueDate())
+                .amount(installmentIngestionFlowFileDTO.getAmount())
+                .debtPositionTypeCode(installmentIngestionFlowFileDTO.getDebtPositionTypeCode())
+                .paymentTypeCode(installmentIngestionFlowFileDTO.getPaymentTypeCode())
+                .remittanceInformation(installmentIngestionFlowFileDTO.getRemittanceInformation())
+                .legacyPaymentMetadata(installmentIngestionFlowFileDTO.getLegacyPaymentMetadata())
+                .flagPagoPaPayment(installmentIngestionFlowFileDTO.getFlagPagoPaPayment())
+                .balance(installmentIngestionFlowFileDTO.getBalance())
+                .flagMultibeneficiary(installmentIngestionFlowFileDTO.getFlagMultiBeneficiary())
+                .numberBeneficiary(installmentIngestionFlowFileDTO.getNumberBeneficiary() != null ? installmentIngestionFlowFileDTO.getNumberBeneficiary() : null)
+                .additionalTransfers(buildTransferList(installmentIngestionFlowFileDTO))
+                .build();
+    }
+
+    private List<TransferSynchronizeDTO> buildTransferList(InstallmentIngestionFlowFileDTO dto) {
+        int nBeneficiary = Optional.ofNullable(dto.getNumberBeneficiary()).orElse(1);
+        if (Boolean.TRUE.equals(dto.getFlagMultiBeneficiary()) && nBeneficiary >= 2) {
+            return IntStream.rangeClosed(2, nBeneficiary)
+                    .mapToObj(index -> createTransfer(dto, index))
+                    .toList();
+        }
+        return List.of();
+    }
+
+
+    private TransferSynchronizeDTO createTransfer(InstallmentIngestionFlowFileDTO dto, int index) {
+        MultiValuedMap<String, String> transferMap = getTransferMapByIndex(dto, index);
+
+        if (transferMap == null) {
+            throw new IllegalStateException("Missing or empty transfer map for index: " + index);
+        }
+
+        return TransferSynchronizeDTO.builder()
+                .orgFiscalCode(getFirstValue(transferMap, "orgFiscalCode"))
+                .orgName(getFirstValue(transferMap, "orgName"))
+                .amount(new BigDecimal(getFirstValue(transferMap, "amount")))
```

Comment:
what about to convert amount into cents and send to debt-position API already as Cents?

@Benedetta-fabbri what do you think about it?